env.example

```diff
@@ -25,6 +25,16 @@ JIBRI_RECORDER_PASSWORD=
 # XMPP password for Jibri client connections
 JIBRI_XMPP_PASSWORD=
 
+#
+# Docker Compose options
+#
+
+# Image tag to use
+IMAGE_TAG=latest
```

Comment:
Can we put this elsewhere, maybe in the compose yml file, so people can update easily? Ideally touching the .env shouldn't be necessary when updating.